package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "yotam-js-metrics-test",
```

Comment:
why it has you name?

Reply:
Because I used it with npm to test the importing and usage. I will change it once we go public with this one.